Expecto/Performance.fs

```diff
@@ -2,53 +2,62 @@
 
 open System
 
-module Performance =
+type SampleStatistics =
+  { N:int
+    mean:float
+    variance:float }
+  member s.standardDeviation = sqrt s.variance
+  member s.meanStandardError = sqrt(s.variance/float s.N)
+
+module internal Statistics =
+
+  let inline private sqr x = x*x
+
+  let initialIntermediateStatistics =
+    0,0.0,0.0
+
+  let inline updateIntermediateStatistics (n,m,s) x =
+    let m'=m+(x-m)/float(n+1)
+    n+1,m',s+(x-m)*(x-m')
+
+  let inline intermediateToSampleStatistics (n,m,s) =
+    {N=n;mean=m;variance=s/float(n-1)}
+
+  /// Online statistics sequence for a given sample sequence.
+  let inline sampleStatistics s =
+    Seq.map float s
+    |> Seq.scan updateIntermediateStatistics initialIntermediateStatistics
+    |> Seq.skip 3
+    |> Seq.map intermediateToSampleStatistics
 
-  type SampleStatistics =
```

Comment:
This used to be inside Performance. Is it major because it exposes a new type? I can maybe put it back how it was.